components/application/logout.jsx

```diff
@@ -22,7 +20,7 @@ const Logout = ({ store }) => {
   return (
     <AppBar.Item className={styles.container}>
       <span className={styles.name}>{name}</span>{' '}
-      <a href={LOGOUT_URL}>Logout</a>
+      <a href={getLogoutUrl(window?.location.origin)}>Logout</a>
```

Comment:
You will need to pass `?reason=logout` to the URL otherwise, logout message doesn't appear.

Reply:
Thank you, I will fix this